src/components/dashboard/index.js

```diff
@@ -178,30 +171,30 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
-const getOrigination = (kind, content, intl) => {
+const getOrigination = (kind, content, t) => {
```

Comment:
Future cleanup: Components could just call `useTranslation` where needed, rather than grabbing `t` from props passed down by a parent component.

I guess we could have already done that with the `injectIntl` HOC, but at least for me it seems more cumbersome to wrap a small component that's not used anywhere else in `injectIntl` 🆚 calling `useTranslation` inside it, especially if wrapping it in an HOC alters the entire component's indentation 😅

Reply:
Actually I did in other spots in this file. I missed it here.